README.md

```diff
@@ -38,8 +38,8 @@ openassetio-traitgen -h
 This package follows the main
 [OpenAssetIO contribution process](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/contributing/PROCESS.md).
 
-However, as a pure Python project, it adheres to strict PEP-8 naming
-conventions.
+However, as a (almost) pure Python project, it adheres to strict PEP-8
```

Comment:
According to [stack exchange](https://english.stackexchange.com/a/115200) you're correct, i.e. the word in brackets doesn't interrupt the sentence, it acts as an adjective, so we should use "an" rather than "a" :grin:

Reply:
Updated in d9bfcdc2